File: API/Orders/Models/OrderDetails.cs
namespace Transflower.Orders.Models;
public class OrderDetails{
    public int Id{get;set;}
    public int MaterialId{get;set;}
    public DateTime OrderDate{get;set;}
    public string? Name{get;set;}
    public string? Category{get;set;}
    public int Quantity{get;set;}
    public int AvailableQuantity{get;set;}
    public string? Status{get;set;}
    public bool ItemStatus{get;set;}
    public string Department{get;set;}    
    public int UserId{get;set;}
    public string ImageUrl{get;set;}

}

File: UI/LatestNGApp/InventoryManagementUI/src/app/Models/orderDetails.ts
export class OrderDetails{

    constructor(

        public id:number,
        public orderDate:Date,
        public status:string,
        public materialId:number,
        public name:string,
        public category:string,
        public quantity:number,
        public availableQuantity:number,
        public department:string,
        public imageUrl:string,
        public itemStatus:boolean,
        public userId:number,

    ){}

}